src/datachain/lib/dc.py

```diff
@@ -859,6 +859,17 @@ def mutate(self, **kwargs) -> "Self":
                 dist=cosine_distance(embedding_text, embedding_image)
         )
         ```
+
+        This method also has enhanced capabilities for renaming the columns. If column
```

Comment:
This method can be also used to rename signals. If the `Column("name")` provided as value for the new signal - the old column will be dropped.

Is it enough to say something like this? ^^

Reply:
Yeah makes sense.